authapi/permissions.py

```diff
@@ -162,3 +172,52 @@ def object_permission_set(self):
                 )
             )
         )
+
+
+class UserPermission(BaseComposedPermision):
+    '''Permissions for the UserViewSet.'''
+    def global_permission_set(self):
+        '''All users must be authenticated. Only admins can create other admin
+        users.'''
+        only_admins_create_admins = Or(
+            AllowAdmin,
+            And(
+                ObjAttrTrue(
+                    lambda r, _: r.data.get('admin').lower() != 'true'),
```

Comment:
So apparently the default `client` sends requests as multipart form, not as JSON. https://github.com/praekelt/seed-auth-api/pull/27/commits/392d6e5fcf4cd5179eddf54a3847ab595adc370f should fix this.